actionpack/lib/action_dispatch/railtie.rb

```diff
@@ -13,12 +13,12 @@ class Railtie < Rails::Railtie # :nodoc:
     config.action_dispatch.default_charset = nil
     config.action_dispatch.rack_cache = false
     config.action_dispatch.http_auth_salt = "http authentication"
-    config.action_dispatch.signed_cookie_salt = "signed cookie"
-    config.action_dispatch.encrypted_cookie_salt = "encrypted cookie"
-    config.action_dispatch.encrypted_signed_cookie_salt = "signed encrypted cookie"
     config.action_dispatch.use_authenticated_cookie_encryption = false
     config.action_dispatch.perform_deep_munge = true
 
+    config.action_dispatch.legacy_encryptor_schemes = []
+    config.action_dispatch.legacy_verifier_schemes  = []
```

Comment:
@kaspth any thoughts on what I should call a complete configuration for a cipher/digest, the salts, and a secret?

I'm thinking I'm just going to call it an `encryptor_config` and `verifiy_config` within the code but am open to something more fitting here!